packages/presentation/lib/screens/login/login_screen.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:presentation/bloc/base/bloc_screen.dart';
+import 'package:presentation/const.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/screens/login/data/login_data.dart';
+import 'package:presentation/screens/login/login_bloc.dart';
+import 'package:presentation/screens/login/widgets/auth_icon_button_widget.dart';
+import 'package:presentation/utils/colors.dart';
+import 'package:presentation/utils/dimensions.dart';
+import 'package:presentation/utils/styles.dart';
+
+class LoginScreen extends StatefulWidget {
+  const LoginScreen({super.key});
+
+  static const _routeName = '/LoginScreen';
+
+  static BasePage page() => BasePage(
+        key: const ValueKey(_routeName),
+        name: _routeName,
+        builder: (_) => const LoginScreen(),
+      );
+
+  @override
+  State<LoginScreen> createState() => _LoginScreenState();
+}
+
+class _LoginScreenState extends BlocScreenState<LoginScreen, LoginBloc> {
+  @override
+  Widget build(BuildContext context) {
+    final appLocalizations = AppLocalizations.of(context)!;
+
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
since the AppBar logic is the same I think we can move it to the App's level as we did it with bottom navigation

Reply:
I am not sure only labels the same on home and login icons, another things are different.

Reply:
agree. but usually, appbar is shared. will discuss it tomorrow I think